addons/account/models/account_reconcile_model.py

```diff
@@ -368,6 +368,24 @@ class AccountReconcileModel(models.Model):
     show_decimal_separator = fields.Boolean(compute='_compute_show_decimal_separator')
     number_entries = fields.Integer(string='Number of entries related to this model', compute='_compute_number_entries')
 
+    def _trigger_auto_reconcile(self, records):
+        # trigger the auto-validate cron on record save if at least one record has auto_reconcile set to True
+        if any(record.auto_reconcile for record in records):
```

Comment:
```suggestion
    def _trigger_auto_reconcile(self):
        # trigger the auto-validate cron on record save if at least one record has auto_reconcile set to True
        if any(self.mapped('auto_reconcile')):
        ...
        # doesn't return anything 
```
No need to pass and return records

```python
def create(self, vals_list):
    records = super().create(vals_list)
    records._trigger_auto_reconcile()
    return records

def write(self, values):
    result = super().write(values)
    self._trigger_auto_reconcile()
    return result # this is usually a boolean not recordset
```

Reply:
fixed !